selfdrive/car/hyundai/values.py

```diff
@@ -1669,6 +1703,9 @@ class Buttons:
 # The camera does SCC on these cars, rather than the radar
 CAMERA_SCC_CAR = {CAR.KONA_EV_2022, }
 
+# These cars have both CAN and CAN-FD messages
+CAN_CANFD_CAR = {CAR.PALISADE_2023, }
```

Comment:
We should be able to detect this, can we add a new flag?

Reply:
Good idea. So far I've only reviewed routes from HDA2 variants and they all have `0x50`, not 100% sure if this message is present on HDA1 or HDA1.5 yet. I can make it so it detects from `0x50` on bus 5 and `0x420` on bus 4.
![image](https://user-images.githubusercontent.com/47793918/219907854-c11fa301-b889-446c-8c21-0cffdf30fa75.png)

Reply:
You can check the message length as well

Reply:
Good to know. I'll see what else I could use. I did this for the time being:
- https://github.com/sunnyhaibin/openpilot-1/commit/baa2724316b2c11205eb66de3887940cf45c3762